client/src/pages/LogIn/LogIn.js

```diff
@@ -26,7 +26,7 @@ class LogIn extends Component {
             applications.
           </h3>
         </div>
-        <br /><br />
```

Comment:
Please remove <br/> anyway. It's recommend not mixing structure (html) and presentation (css)